File: rhino-experiment/src/main/resources/calculator-service.js
var calculatorService = (function() {
    var addNumbers = function(a, b) {
        return a + b;
    };

    var subNumbers = function(a, b) {
        return a - b;
    };

    var mulNumbers = function(a, b) {
        return a * b;
    };

    var divNumbers = function(a, b) {
        return a / b;
    };

    return {
        "addNumbers": addNumbers,
        "subNumbers": subNumbers,
        "mulNumbers": mulNumbers,
        "divNumbers": divNumbers
    };
})();

File: qdox-experiment/src/main/java/me/loki2302/CalculatorService.java
package me.loki2302;


import java.util.function.Function;

/**
 * Implements calculator functionality
 */
public class CalculatorService {
    /**
     * @param a number A
     * @param b number B
     * @return sum of a and b
     */
    public int addNumbers(int a, int b) {
        return a + b;
    }

    public int someJava8Stuff() {
        Function<Integer, Integer> f = a -> a + 1;
        return f.apply(2);
    }
}
